scripts/aws-sso/aws-sso-login.sh

```diff
@@ -33,16 +33,18 @@ function log {
 SCRIPT_LOG_LEVEL=${SCRIPT_LOG_LEVEL:-2}
 PROJECT=$(hcledit -f "$COMMON_CONFIG_FILE" attribute get project | sed 's/"//g')
 SSO_PROFILE_NAME=${SSO_PROFILE_NAME:-$PROJECT-sso}
-SSO_ROLE_NAME=${SSO_ROLE_NAME:-$(hcledit -f "$ACCOUNT_CONFIG_FILE" attribute get sso_role | sed 's/"//g')}
+#SSO_ROLE_NAME=${SSO_ROLE_NAME:-$(hcledit -f "$ACCOUNT_CONFIG_FILE" attribute get sso_role | sed 's/"//g')}
```

Comment:
I guess we could drop these commented lines, since they only end up cluttering the code. We can always rely on git's history to recover them in case we realize they were needed.